### PR TITLE
HomePage

### PR DESCRIPTION
**Pull request by jw823**
Made the Homepage

I still need to consolidate to make it more concise, and there are a few formatting issues. For the most part, the layout is there.

Screen Shot 2020-02-17 at 8 57 19 PM

### CODE REVIEW

File: lib/Current_Ride.dart [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+            child: Row(
+              children: <Widget>[
+                Padding(
+                  padding: const EdgeInsets.only(top: 8.0),
+                  child: Icon(Icons.person_pin, size: 38.0),
+                ),
+                SizedBox(width: 5.0),
+                Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  //mainAxisAlignment: MainAxisAlignment.spaceEvenly,
```
Comment (SamChou19815) [inline lib/Current_Ride.dart]: Remove commented code

File: lib/Upcoming_Ride.dart [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+            width: 260.0,
+            height: 300.0,
+            child: Card(
+              elevation: 3.0,
+              child: Column(
+                children: <Widget>[
+                  Container(
+                    margin: EdgeInsets.symmetric(horizontal: 15.0),
+                    child: Row(
+                      //mainAxisAlignment: MainAxisAlignment.spaceEvenly,
```
Comment (SamChou19815) [inline lib/Upcoming_Ride.dart]: No commented code

**Comment by SamChou19815** [comment]
@jw823 Could you also resolve the merge conflict against master?

**Comment by jw823** [comment]
> Could you also resolve the merge conflict against master?

I need to meet with Tony for that. Most likely before stand-ups on Wednesday or earlier

File: lib/Ride_History.dart [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+        fontSize: 17,
+        height: 2
+    );
+
+    return Column(
+      children: <Widget>[
+        Container(
+          margin: EdgeInsets.only(left: 17.0, right: 17.0, bottom: 15.0),
+          width: 400.0,
+          height: 125.0,
```
Comment (chansen424) [inline lib/Ride_History.dart]: Do not set absolute size. Content will overflow containers. Instead use relative sizing by using MediaQuery.of(context).size.width and .height

File: lib/Ride_History.dart [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+                  ),
+                )
+              ],
+            ),
+          ),
+        ),
+        Container(
+          margin: EdgeInsets.only(left: 17.0, right: 17.0, bottom: 15.0),
+          width: 400.0,
+          height: 125.0,
```
Comment (chansen424) [inline lib/Ride_History.dart]: Absolute sizing should be relative

File: lib/Upcoming_Ride.dart [hunk 5]
```diff
@@ -0,0 +1,278 @@
...
+    return Container(
+      margin: EdgeInsets.symmetric(horizontal: 17.0),
+      height: 260.0,
+      child: ListView(
+        scrollDirection: Axis.horizontal,
+        children: <Widget>[
+          Container(
+            margin: EdgeInsets.only(right: 15.0),
+            width: 260.0,
+            height: 300.0,
```
Comment (chansen424) [inline lib/Upcoming_Ride.dart]: Absolute sizing should be relative

File: lib/Home.dart [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+    return Scaffold(
+      endDrawer: Drawer(
+        child: ListView(
+          padding: EdgeInsets.all(5.0),
+          children: <Widget>[
+            ListTile(
+              leading: Icon(Icons.person),
+              title: Text('Profile'),
+              onTap: (){
+                Navigator.pop(context);
```
Comment (chansen424) [inline lib/Home.dart]: Do not pop. Instead push the Profile page that Tony made to the Navigator

File: lib/Home.dart [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+              },
+            ),
+            Divider(
+              color: Colors.grey[500],
+            ),
+            ListTile(
+              leading: Icon(Icons.settings),
+              title: Text('Settings'),
+              onTap: (){
+                Navigator.pop(context);
```
Comment (chansen424) [inline lib/Home.dart:36]: Don't pop, instead Navigate to Tony's Settings page, which is now merged.

File: lib/Home.dart [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+              },
+            ),
+            Divider(
+              color: Colors.grey[500],
+            ),
+            ListTile(
+              leading: Icon(Icons.notifications),
+              title: Text('Notifications'),
+              onTap: (){
+                Navigator.pop(context);
```
Comment (chansen424) [inline lib/Home.dart:46]: This is fine for now since pop() has no effect on the Homepage.

**Pull request merged**